SETUP/upgrade/19/20230627_alter_tasks.php

```diff
@@ -0,0 +1,21 @@
+<?php
+$relPath = '../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Delete task_version column from tasks table\n";
+$sql = "
+    ALTER TABLE tasks
+        DROP COLUMN task_version
+";
+
+echo "$sql\n";
+
+mysqli_query(DPDatabase::get_connection(), $sql) or die(mysqli_error(DPDatabase::get_connection()));
```

Comment:
You'll want to update the initial create schema as well. https://github.com/DistributedProofreaders/dproofreaders/blob/aae0d560d23a110d3489eee7d490decd4bb3f9fb/SETUP/db_schema.sql#L546

Reply:
Thanks @chrismiceli